yoga/Yoga.c

```diff
@@ -179,10 +179,26 @@ static inline float YGComputedEdgeValue(const float edges[YGEdgeCount],
   }
 
   if (edge == YGEdgeStart || edge == YGEdgeEnd) {
-    return YGUndefined;
+    YGValue result;
+    result.value = YGUndefined;
+    result.defined = false;
+    result.unit = YGUnitPixel;
+    return result;
   }
 
-  return defaultValue;
+  YGValue result;
+  result.value = defaultValue;
+  result.defined = (defaultValue < 0 || defaultValue >= 0); /* is faster than a nan function call and enough at this point */
+  result.unit = YGUnitPixel;
+  return result;
```

Comment:
Please don't let me use this syntax here and at the other spots. Visual Studio (at least VS13 which I use) doesn't recognize this syntax, which would need me to change this one everytime I need to compile. (thats the reason I missed some tests at first)